MdeModulePkg/Universal/SetupBrowserDxe/Setup.c

```
@@ -3632,8 +3716,10 @@ SubmitForFormSet (
         gCurrentSelection->Action = UI_ACTION_REFRESH_FORMSET;
         gCurrentSelection->Handle = FormSet->HiiHandle;
         CopyGuid (&gCurrentSelection->FormSetGuid, &FormSet->Guid);
-        gCurrentSelection->FormId     = Form->FormId;
-        gCurrentSelection->QuestionId = Question->QuestionId;
+        gCurrentSelection->FormId = Form->FormId;
+        if (Question != NULL) {

```

It's not clear to me that ignoring this error is safe. Please return an error.